Python/Product/Debugger/Debugger/PythonProcess.cs

```diff
@@ -488,6 +488,21 @@ public PythonProcess(PythonLanguageVersion languageVersion, string exe, string a
             return ast;
         }
 
+        internal SourceSpan GetStatementSpan(string filename, int line, int column = 0) {
```

Comment:
Remove column param that is unused?

Reply:
Planning for future use - if we switch to an AST walk (which might make sense) and start doing opcode lookups then the column will be relevant. (Might not be available, but we'll deal with that when it comes.)